cinema/models.py

```diff
@@ -1,9 +1,26 @@
 from django.db import models
 
 
+class Genre(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+
+
+class Actor(models.Model):
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+
+
+class CinemaHall (models.Model):
+    name = models.CharField(max_length=255)
+    rows = models.IntegerField()
+    seats_in_row = models.IntegerField()
+
+
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="actors")
+    genres = models.ManyToManyField(Genre, related_name="genres")
```

Comment:
The `related_name` for the `genres` field should be something like `movies` instead of `genres`. This is because `related_name` is used to access the reverse relation from the `Genre` model to the `Movie` model, and `movies` would be more intuitive.